assets/css/custom.scss

```diff
@@ -136,35 +136,12 @@ blockquote {
     text-align: center;
     margin-bottom: $spacer-3;
     line-height: 1.25;
+    margin-top: $spacer-6;
 
-    &:before {
-      display: block;
-      font-size: 17px;
-      font-weight: 700;
-      margin-bottom: $spacer-4;
-      margin-top: 40px;
-
-      @include breakpoint(md) {
-        margin-top: 120px;
-      }
-
-    }
-  }
-
-  h2:nth-of-type(1):before {
-    content: "Section 1";
     @include breakpoint(md) {
-      margin-top: 60px;
+      margin-top: $spacer-6 * 2;
     }
   }
-  h2:nth-of-type(2):before { content: "Section 2"; }
```

Comment:
TIL: https://www.smashingmagazine.com/2014/06/css-driven-internationalization-in-javascript/#css-driven-internationalization-improved